core/src/main/java/org/apache/accumulo/core/iterators/ColumnFamilyCounter.java

```diff
@@ -89,4 +89,11 @@ public Value getTopValue() {
     return null;
   }
 
+  @Override
```

Comment:
I would definitely be interested in clean up of the iterators.  @ctubbsii @joshelser Any ones in particular you had in mind?  I will create a ticket.